packages/core/react/use-chat.ts

```diff
@@ -343,6 +357,20 @@ export function useChat({
       } finally {
         mutateLoading(false);
       }
+
+      // auto-submit when all tool calls in the last assistant message have results:
+      const lastMessage = messagesRef.current[messagesRef.current.length - 1];
```

Comment:
`lastMessage` can't be undefined here, right?

Reply:
it should, bc it comes after a message has been received, but worth adding a check